plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,189 @@
+from proxmoxer import ProxmoxAPI
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.proxmox import (
+    proxmox_auth_argument_spec, ProxmoxAnsible, proxmox_to_ansible_bool)
+
+import urllib3
+
+urllib3.disable_warnings()
+
+DOCUMENTATION = '''
+---
+module: proxmox_backup_info
+short_description: Retrieve information about one or more Proxmox VE backups
+version_added: 2.2.0
+description:
+  - Retrieve information about one or more Proxmox VE backups.
+options:
+  id:
+    description:
+      - Only return information on a specific backup.
+    type: str
+author: Leverrier Dylan (@tleguern)
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+notes:
+  - Backup specific options can be returned by this module, please look at the documentation at U(https://pve.proxmox.com/wiki/Backup_and_Restore).
+'''
+
+EXAMPLES = '''
+- name: List existing backups
+  community.general.proxmox_backup_info:
+    api_host: helldorado
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+    validate_certs: "{{ validate_certs | default(omit) }}"
+    id: "{{ id | default(omit) }}"
+  register: proxmox_backups
+'''
+
+RETURN = '''
+proxmox_backups:
+  description: List of backup job.
+  returned: on success
+  type: list
+  elements: dict
+  contains:
+    mailnotification:
+      description: Notification event type
+      returned: on success
+      type: str
+    storage:
+      description: Storage used in backup job
+      returned: on success
+      type: str
+    notes-template:
+      description: Template string for generating notes for the backup(s).
+      returned: on success
+      type: str
+    mailto:
+      description: Comma-separated list of email addresses or users that should receive email notifications.
+      returned: on success
+      type: str
+    id:
+      description: Job ID.
+      returned: on success
+      type: str
+    mode:
+      description: Backup mode
```

Comment:
```suggestion
      description: Backup mode.
```

Reply:
Fix in commit [49d6df9541b7e86c582d5b342f3d1677f4523f5b](https://github.com/ansible-collections/community.general/pull/8301/commits/49d6df9541b7e86c582d5b342f3d1677f4523f5b)